Add october newsletter (#3)
YunoHost/news: 2019/2019-10.md

![YunoHost Project logo](../img/yunohost.png)

# YunoHost Project Newsletter

_October 2019_

## 3.6.5.1 released

> A new minor release has been made for YunoHost 3.6.5 series including some much needed fixes for issues that were plaguing users.

Reviewer: ```suggestion
A new minor release has been made for the YunoHost 3.6.5 series, including some much needed fixes for some user plaguing issues.
```

[Read more.](https://forum.yunohost.org/t/yunohost-3-6-release-sortie-de-yunohost-3-6/8359/61)

## 3.7 testing release

> A new testing release is available for early adopters and enthusiasts who would like to support the development team to release the new release. This release has some interesting features such as group and permissions mechanisms, application installation fixes, removal of Etckeeper and a whole lot of translation work. Thanks so much to all the contributors!

Reviewer: ```suggestion
A new testing release is available for early adopters and enthusiasts who would like to support the development team release the new YunoHost. You might find interesting features such as group and permissions mechanisms, application installation fixes, removal of Etckeeper and a whole lot of translation work. Thanks so much to all the contributors!
```

[Read more.](https://forum.yunohost.org/t/yunohost-3-7-spooky-testing-call-for-feedback/9385)

## Progress on Debian Buster support

> A big shout out goes to the contributors who have been supporting the development work towards enabling YunoHost to run on the latest Debian Buster release. The work is progressing but there is still more to do, please see the ticket below and help out if you have time to spare.

Reviewer: ```suggestion
A big shout-out to the contributors working towards having YunoHost run on the latest Debian Buster. The development is progressing, but there is still more to do, please see the ticket below and help out if you have time to spare.
```

[Read more.](https://github.com/YunoHost/issues/issues/1255)

## YunoHost stand at WebAssoc

> YunoHost users held a stand promoting the project and discussing and showing demos on how to self-host at the latest [webassoc.org](https://www.webassoc.org/) event. If you are also considering giving a talk on YunoHost, please feel free to do so! There are some [slides (FR)](https://gitprint.com/alexAubin/yunohostWorkshop/blob/master/sheet_fr.md) and a [workshop guide](https://github.com/alexAubin/yunohostWorkshop) to help you get started!

Reviewer: ```suggestion
YunoHost users held a stand promoting the project, discussing and presenting demos on how to self-host at the latest [webassoc.org](https://www.webassoc.org/) event. If you are also considering giving a talk on YunoHost, please feel free to do so! Here are some [slides (FR)](https://gitprint.com/alexAubin/yunohostWorkshop/blob/master/sheet_fr.md) and a [workshop guide](https://github.com/alexAubin/yunohostWorkshop) to help you get started!
```

[Read more.](https://www.webassoc.org/events/libre-191119/)

## Project donation platform launched

> The YunoHost core team have launched a platform to allow the community to support the work in order to make the project more financially sustainable.  Furthermore, the platform is packaged as [a YunoHost application](https://github.com/YunoHost-Apps/transpay_ynh)! You can use it for your own purposes too.

Reviewer: ```suggestion
The YunoHost core team have launched a funding platform to allow the community to support YunoHost financially. Furthermore, the platform is packaged as [a YunoHost application](https://github.com/YunoHost-Apps/transpay_ynh)! You can use it for your own project too.
```

[Read more.](https://donate.yunohost.org/)

## Users speak out

* [Installing Discourse in a 5 minute demo](https://social.logilab.org/@arthurlutz/102904482336325351)

## YunoHost at FOSDEM 2020

> [@decentral1se](https://github.com/decentral1se) (YunoHost contributor) has been asked to join the steering committe for the decentralized internet & privacy devroom at FOSDEM 2020. Some sort of workshop or talk proposal is in progress so expect to hear more about this before February.

Reviewer: ```suggestion
[@decentral1se](https://github.com/decentral1se) (YunoHost contributor) has been asked to join the steering committe for the Decentralized Internet and Privacy devroom FOSDEM in 2020. Some sort of workshop or talk proposal is in progress, so expect more news on this before February.
```

[Read more.](https://lists.fosdem.org/pipermail/fosdem/2019q4/002903.html)

## YunoHost interview for Radiostudent.si

[@alexAubin](https://github.com/alexAubin) (YunoHost core) was asked to participate in an interview about the project for the [radiostudent.si] initiative. The majority of the podcast is in Slovenian.

[Read more.](https://radiostudent.si/dru%C5%BEba/tehno-klistir/sam-svoj-gostitelj)

## A new home page for YunoHost

> There has been some great work done in order to re-imagine a new landing page for YunoHost over at [eliegavoty.fr/testou/fr](https://eliegavoty.fr/testou/fr). However, more energy is needed and we are looking for anyone who would like to help! The source code is available [here](https://github.com/e-lie/yuno-landpage-grav). Please [come and chat to us](https://yunohost.org/#/chat_rooms_en) if you would like to help.

Reviewer: ```suggestion
Some great work in re-imagining a new landing page for YunoHost is to be found over at [eliegavoty.fr/testou/fr](https://eliegavoty.fr/testou/fr). However, more energy is needed, and we are looking for people wanting to help! The source code is available [here](https://github.com/e-lie/yuno-landpage-grav). Please [come and chat to us](https://yunohost.org/#/chat_rooms_en) if you would like to help.
```

[Read more.](https://github.com/e-lie/yuno-landpage-grav)

## New developer environment

> A lot of internal work has gone towards enabling a simpler and easier workflow for YunoHost contributors. This has always been a source of pain for new contributors and a barrier to entry. The process of setting up a development environment for YunoHost has been dramatically simplified and it's never been simpler to contribute to the development of the project!

Reviewer: ```suggestion
A lot of internal work has targeted simpler and easier contribution workflow. This barrier to entry has always been a source of pain for new contributors. The process of setting up a development environment for YunoHost has been dramatically simplified, it's never been simpler to contribute to the development of the project!
```

[Read more.](https://github.com/YunoHost/ynh-dev#table-of-contents)

## Podcasts from @unklebonehead

> [@unklebonehead](https://github.com/unklebonehead) has been hard at work releasing solid podcasts (under the title of "The Self Hosted Podcast") about how to work with the YunoHost project in various ways. As self-described: "Started as a way for me to showcase Yunohost but is quickly evolving into everything that encompasses the fediverse.". They come well recommended and are available to listen to and load from your favourite podcasting free software today.

Reviewer: ```suggestion
[@unklebonehead](https://github.com/unklebonehead) has been hard at work releasing solid podcasts (under the title of "The Self Hosted Podcast") about how to work with the YunoHost project in various ways. As self-described: "Started as a way for me to showcase YunoHost, but quickly evolving into everything that encompasses the Fediverse.". They come recommended and are available for your listening pleasure from your favourite free software music player today.
```
